src/externalized/DefaultContentManager.cc

```diff
@@ -1333,6 +1354,41 @@ const VehicleModel* DefaultContentManager::getVehicle(uint8_t const vehicleID) c
 	return m_vehicles[vehicleID];
 }
 
+NPCQuoteInfo* DefaultContentManager::getScriptRecords(uint8_t profileId) const
+{
+	MercProfile profile(profileId);
+	ST::string filename;
+	if (profile.isPlayerMerc() || (profile.isRPC() && profile.isRecruited()))
+	{
+		filename = "script-records-recruited.json";
+		auto json = readJsonDataFile(filename);
+		return NPCQuoteInfo::deserialize(json.toVec()[0], this, this);
+	}
+	else
+	{
+		return (NPCQuoteInfo*)m_scriptRecords[profileId];
```

Comment:
This cast is needed because m_scriptRecords is a vector of NPCQuoteInfo const *, right? Ideally all function's returning NPQuoteInfo * should be checked if they should not return a pointer to const instead but if you don't want to do this now please use a const_cast here.